_docs/tasks/traffic-management/services-expansion.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Accessing Services in the Expanded Mesh
+overview: This task shows you how to use services provided by VM
+
+order: 60
+
+#draft: true
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows you how to configure services running in a VM that joined the cluster.
+
+Current task was tested on GCP. _WIP on adding specific info for other providers_
```

Comment:
The example in this task was tested on Google Cloud Platform (GCP).

(presume it's on Compute Engine?)